src/Promenade.Models/Entities/SegmentText.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Ocuda.Promenade.Models.Abstract;
+
+namespace Ocuda.Promenade.Models.Entities
+{
+    public class SegmentText : BaseEntity
```

Comment:
This should not extend BaseEntity as it should have a composite key of SegmentId and LanguageId (unless this ties in with work @iafb is doing on model refactoring)